scripts/skills/perks/perk_rf_battle_fervor.nut

```diff
@@ -1,9 +1,10 @@
 this.perk_rf_battle_fervor <- ::inherit("scripts/skills/skill", {
 	m = {
 		Stacks = 0,
+		HasAttackedThisTurn = false,
 
 		MaxStacks = 4,
-		MultPerStack = 0.05
```

Comment:
This renaming must also happen down below in the `RF_getMult` function

Reply:
It's already done, you might be looking at an older version of the code.